.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file should not have been included in the pull request. I see that you've added the `.idea/` folder to the `.gitignore` file, but apparently you must've done this after it had already pushed some stuff from it.